packages/nds-stacked-nav/src/StackedNav.js

```diff
@@ -3,31 +3,103 @@ import PropTypes from "prop-types";
 
 import "./../scss/stacked-nav.scss";
 
-export const StackedNav = props => {
+const Heading = props => {
+	const { elementType: ElementType = "p", label, link } = props;
+	if (link) {
+		const { elementType: LinkElementType = "a", isCurrent, destination } = link;
+		const linkProps = {
+			"aria-current": isCurrent ? "true" : "false"
+		};
+		if (LinkElementType === "a") {
+			linkProps.href = destination;
+		} else {
+			linkProps.to = destination;
+		}
+		return (
+			<ElementType className="stacked-nav__root">
+				<LinkElementType {...linkProps}>{label}</LinkElementType>
+			</ElementType>
+		);
+	}
 	return (
-		<nav className="stacked-nav">
-			<h2 className="stacked-nav__root">
-				<a href="#">Root link</a>
-			</h2>
-			<ul className="stacked-nav__list">
-				<li className="stacked-nav__list-item">
-					<a href="#">Link 1</a>
-				</li>
-				<li className="stacked-nav__list-item">
-					<a href="#" aria-current="page">
-						Link 2
-					</a>
-				</li>
-				<li className="stacked-nav__list-item">
-					<a href="#">Link 3</a>
-				</li>
-			</ul>
-		</nav>
+		<ElementType className="stacked-nav__root stacked-nav__root--no-link">
+			{label}
+		</ElementType>
 	);
 };
 
-StackedNav.propTypes = {};
+Heading.propTypes = {
+	label: PropTypes.node,
+	elementType: PropTypes.elementType,
+	link: PropTypes.shape({
+		destination: PropTypes.string,
+		isCurrent: PropTypes.bool,
+		elementType: PropTypes.elementType
+	})
+};
+
+export const StackedNavLink = props => {
+	const {
+		hint,
+		destination,
+		isCurrent,
+		elementType: ElementType = "a",
+		children
+	} = props;
+	const label = props.label || children;
+	if (!label) return null;
+	const linkProps = {};
+	linkProps["aria-current"] = isCurrent ? "true" : "false";
+	if (ElementType === "a") {
+		linkProps.href = destination;
+	} else if (ElementType) {
+		linkProps.to = destination;
+	}
+	return (
+		<li className="stacked-nav__list-item">
+			<ElementType {...linkProps}>
+				{hint ? (
+					<>
+						{label}
+						<span className="stacked-nav__hint">{hint}</span>
+					</>
+				) : (
+					label
+				)}
+			</ElementType>
+		</li>
+	);
+};
 
-StackedNav.defaultProps = {};
+StackedNavLink.propTypes = {
+	destination: PropTypes.node,
```

Comment:
Shouldn't this be PropTypes.string?

```suggestion
	destination: PropTypes.string,
```